python/cuml/test/conftest.py

```diff
@@ -211,68 +221,3 @@ def nlp_20news():
     Y = cp.array(twenty_train.target)
 
     return X, Y
-
-
-def pytest_addoption(parser):
-    parser.addoption("--run_stress",
-                     action="store_true",
-                     default=False,
-                     help="run stress tests")
-
-    parser.addoption("--run_quality",
-                     action="store_true",
-                     default=False,
-                     help="run quality tests")
-
-    parser.addoption("--run_unit",
-                     action="store_true",
-                     default=False,
-                     help="run unit tests")
-
-
-def pytest_collection_modifyitems(config, items):
-    if config.getoption("--run_quality"):
-        # --run_quality given in cli: do not skip quality tests
-        skip_stress = pytest.mark.skip(
-            reason="Stress tests run with --run_stress flag.")
-        for item in items:
-            if "stress" in item.keywords:
-                item.add_marker(skip_stress)
-        skip_unit = pytest.mark.skip(
-            reason="Stress tests run with --run_unit flag.")
-        for item in items:
-            if "unit" in item.keywords:
-                item.add_marker(skip_unit)
-
-        return
-
-    else:
-        skip_quality = pytest.mark.skip(
-            reason="Quality tests run with --run_quality flag.")
-        for item in items:
-            if "quality" in item.keywords:
-                item.add_marker(skip_quality)
-
-    if config.getoption("--run_stress"):
-        # --run_stress given in cli: do not skip stress tests
-
-        skip_unit = pytest.mark.skip(
-            reason="Stress tests run with --run_unit flag.")
-        for item in items:
-            if "unit" in item.keywords:
-                item.add_marker(skip_unit)
-
-        skip_quality = pytest.mark.skip(
-            reason="Quality tests run with --run_quality flag.")
-        for item in items:
-            if "quality" in item.keywords:
-                item.add_marker(skip_quality)
-
-        return
-
-    else:
-        skip_stress = pytest.mark.skip(
-            reason="Stress tests run with --run_stress flag.")
-        for item in items:
-            if "stress" in item.keywords:
-                item.add_marker(skip_stress)
```

Comment:
@mdemoret-nv this seems to have been checked in as part of other work re-organizing pytest related to issue #3093? This seems to be the culprit of running the stress tests in CI unless I'm missing another change that could be the reason?